src/resources/asres.rs

```diff
@@ -19,6 +19,7 @@ use std::str::FromStr;
 use bcder::{decode, encode};
 use bcder::Tag;
 use bcder::encode::PrimitiveContent;
+use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
```

Comment:
Matter of style: I keep module imports and type imports on separate lines, i.e.,

```
use serde::de;
use serde::{Deserialize, ...};
```

Reply:
pushed